ompi/communicator/communicator.h

```diff
@@ -194,6 +218,19 @@ struct ompi_communicator_t {
      * collective coexists using multiple backends).
      */
     opal_atomic_int32_t c_nbc_tag;
+
+#if OPAL_ENABLE_FT_MPI
```

Comment:
This change does not move existing fields, and due to padding in predefined should not cause ABI compatibility issues.

Reply:
But given that we're already breaking ABI between 4.x and 5.0, should we add more padding?

Reply:
There is 48 bytes left with LP64.